python/GangaLHCb/test/GPI/TestExternalGaudiExec.py

```diff
@@ -110,6 +110,10 @@ def testSubmitJobComplete(self):
         from Ganga.GPI import jobs
 
         j = jobs[-1]
+        j = j.copy()
+        j.unprepare()
+        j.prepare()
+        j.submit()
```

Comment:
@rob-c The PR #980 has a test case implemented that shows two problems with the shared area implementation. Discussion on that topic should continue there.